src/shared/cards/Card Template.js

```diff
@@ -0,0 +1,76 @@
+import Card from 'cards/Card';
+
+export default class CardName extends Card {
+  static title = 'Card Name';
+  cost = 4;
+  VP = 3;
+  coin = 2;
+  types = new Set(['Action', 'Treasure', 'Attack', 'Reaction', 'Victory']);
+  async onPlay(player) {
+    // Vanilla Effects
+    player.draw(1);
```

Comment:
I think you'll have to do `await player.draw(1)`
same for `player.trash`, `player.play`, `player.discard`, etc